variables.tf

```
@@ -472,3 +472,17 @@ variable "skip_ocp_secrets_manager_iam_auth_policy" {
   description = "To skip creating auth policy that allows OCP cluster 'Manager' role access in the existing Secrets Manager instance for managing ingress certificates."
   default     = false
 }
+
+variable "install_dependencies" {
```

perhaps this is better?

```suggestion
variable "install_required_binaries" {
```